packages/abi-typegen/test/fixtures/templates/script/main.hbs

```diff
@@ -1,7 +1,8 @@
-/* Autogenerated file. Do not edit manually. */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/consistent-type-imports */
 
-/* tslint:disable */
-/* eslint-disable */
+/* Autogenerated file. Do not edit manually. */
```

Comment:
I think this message should still be at the top of the file.

Reply:
c32118749f8c99723ac7f094aeb14a716dbee117